app/frontend/src/components/Markdown.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { increaseMarkdownHeaderLevel } from "./Markdown";
+
+describe("markdown header level increase", () => {
+  it.each`
+    source                                           | topLevel | result
+    ${"#hi\n##hello\nYo\n###omg"}                    | ${3}     | ${"###hi\n####hello\nYo\n#####omg"}
+    ${"#hi\n##hello\nYo\n###omg"}                    | ${1}     | ${"#hi\n##hello\nYo\n###omg"}
+    ${"#hi\n##hello\nYo\n###omg"}                    | ${6}     | ${"######hi\n######hello\nYo\n######omg"}
+    ${"#h1\n##h2\n###h3\n####h4\n#####h5\n######h6"} | ${3}     | ${"###h1\n####h2\n#####h3\n######h4\n######h5\n######h6"}
```

Comment:
Might be worth testing all `topLevel` from 1-6 if you are doing this? I can see 2 and 4 missing in this